recOrder/io/zarr_converter.py

```diff
@@ -19,31 +18,41 @@ class ZarrConverter:
     for tiled acquisitions)
     """
 
-    def __init__(self, input, output, data_type=None, replace_position_names=False, format_hcs=False):
+    def __init__(self, input_dir, output_dir, data_type=None, replace_position_names=False, format_hcs=False):
```

Comment:
Much clearer name choice. 

I think the user-facing command-line options should match this choice, but that will break backwards compatibility, so I'll mark it as a 1.0.0 change.

Reply:
See #147